README.md

```diff
@@ -125,3 +125,7 @@ providing Insights(security, licenses, AI based guidance) for applications and h
 # License
 
 Apache 2.0, See [LICENSE](LICENSE) for more information.
+
+# Data and telemetry
+
+The Red Hat Dependency Analytics Extension for Visual Studio Code collects anonymous usage data and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.elemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-commons#how-to-disable-telemetry-reporting
```

Comment:
you should also link to a page detailing what data is being collected by CRDA

Reply:
added an MD file (https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/486/files#diff-15fb298badf01968e9dc0621fd4bee9f931243ad8fb1c52dbaa3bd37764453a0)